cluster/gce/gci/mounter/stage-upload.sh

```diff
@@ -20,7 +20,6 @@
 # put in configure.sh, which is uploaded via GCE custom metadata.
 
 set -o errexit
-set -o pipefail
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2039

We can switch the shebang to bash instead of sh instead of removing this line (I think I am in favor of that). The image used include bash as well.

Reply:
+1, let's switch to bash.